### ISSUE TITLE
Is this project still actively developed?

### ISSUE THREAD

**Issue by jasonkhanlar** [arc298/instagram-scraper#678]
Is this project still actively developed? There are bugs for 1+ year that have made the app not functional for many uses. If inactive, since this is project is in public domain, is it possible that someone else could take over maintaining latest revision?

Looking at https://github.com/arc298/instagram-scraper/network showing the 100 most recently pushed forks:

@benwinding 3 days ago

@speedholicktp 24 days ago

@ShuBhamg0sain 1 month ago

@duskvirkus 7 months ago

@Ahmedasdube 7 months ago

the rest are 1+ year ago

Note: I'm not sure on procedure for this, but I was just curious to mention because it appears @arc298 hasn't been active since 7 months

**Comment by jasonkhanlar** [comment]
https://github.com/arc298/instagram-scraper/issues/578 appears to be the last issue that arc298 was active

**Comment by jasonkhanlar** [comment]
@benwinding, I glanced briefly at each of the users I listed above and you seem to be the most active and personally I think you would be a most reliable/reputable to continue/maintain this project. What do you think?

**Comment by VasiliPupkin256** [comment]
Why have you arranged contributors by the last commit and ignored the amount of contribution? @benwinding have only made one trivial commit and filed a single pull request that wasn't even accepted. I don't think he even understand the codebase well. @AlexNik contributed much more and you don't even list him.

**Comment by AlexNik** [comment]
@VasiliPupkin256 Thanks))
I think we should try to found some contacts of @arc298 and ask him to "Transfer ownership" of this repo to somebody...

**Comment by VasiliPupkin256** [comment]
By the way. The other most active contributors who have authored of at least 60 lines of code are also @orkaa @Skinner927 @Aaronzinhoo @Emalton @laranicolas @lambriggerbrian and none of them have listed by you.

**Comment by Aaronzinhoo** [comment]
I agree we should ask for transfer of ownership. It's been hard to make consistent commits when we don't know when they may be merged. Either that or a fork of this repo!

**Comment by VasiliPupkin256** [comment]
I see the major problem of the project is not the lack of support from the @arc298 but the efficient instagram measures against scrapping. Neither instagram-scraper nor instaloader or any other popular instagram downloaders are able to work nowadays. Profiles are banned for scraping. I've had an authorized profile which was flagged for suspicious activity and the phone number which it was registered on is now blacklisted on instagram. You can only scrape one medium sized instagram profile, or download sctories of like 15 profiles, before you would be flagged for suspicious activity. 

Maintainers of instagram downloaders are either abandoned their projects or ignoring the issue, I was banned by @aandergr in the instaloader repo for a single comment about the issue here https://github.com/instaloader/instaloader/issues/834 they choose to ignore the elephant in the room and Richard Arcega has abandoned the current project probably because of this also. You just can't support the project that is causing profile bans for the users who try to use it.

**Comment by arc298** [comment]
Hey, I'm still around :) I apologize for the lack of activity--sometimes life throws you some curveballs. I'm certainly open to bringing on collaborators who are interested in maintaining the project or improving it for the benefit of the community. 

I would welcome anyone who fits this criteria:

- Have contributed a non-trivial amount of code to the repo
- Active and enthusiastic about improving the app
- Enjoy using this app  :)

You can be as much or as little involved as you want. The responsibilities can include any of the following:

- Improving documentation
- Responding to issues
- Reviewing and merging pull requests
- Development contributions: improving code quality, adding features, fixing bugs
- Establish and manage a release process

**Comment by AlexNik** [comment]
I'd like to be a collaborator)
I've had some PR already)

**Comment by Aaronzinhoo** [comment]
@arc298  thanks for the invite as a collaborator. I believe we should create something of a feature wish list moving forward. There has been a lot of changes by Instagram in terms of security recently so we can also see which portions of the code base are able to be used reasonably without getting banned. 

I think setting up a smooth docker container deployment should also be high on the list! Not too hard and I can get started on it soon. any thoughts ? @arc298 @AlexNik @VasiliPupkin256

**Comment by AlexNik** [comment]
@Aaronzinhoo I've alredy done it 😄 
See #659 #660

**Comment by Aaronzinhoo** [comment]
Haha that's perfect! Then you got any other ideas? I do think having a discussion feature wish list would be pretty nice. It would be a nice way to interact with the community! @AlexNik

**Comment by VasiliPupkin256** [comment]
How about solving the actual issue that blocks all the others? The account bans for suspicious activity? How will you use the docker? Would you create or buy trillion new accounts and sim cards every day or casually scrape stories for a couple of accounts only. I just don't know how to proceed. It is pain to even debug the tool when every new account is banned in a process.

**Comment by AlexNik** [comment]
Docker is just the simple way for run this app without any dependencies.
Without python or other.
The problem with banning accounts is the separate problem.

**Comment by FatemeKhodayari** [comment]
@arc298 I've found and used this project recently (just a week or so) and I've been in trouble with lack of enough documentation and example. I'd like to help if it's ok with you :)

**Comment by AlexNik** [comment]
@arc298 Could you release v1.10.0 to PyPi?
Could you send me email, plz? I want to discuss some issues.
You could find my email in my profile.

**Comment by ShuBhamg0sain** [comment]
Yes
